db/migrate/20170519153350_add_showback_models.rb

```diff
@@ -0,0 +1,73 @@
+class AddShowbackModels < ActiveRecord::Migration[5.0]
+  def up
+    create_table :showback_usage_types, :id => :bigserial, :force => :cascade do |t|
+      t.string     :category, :allow_nil => false
+      t.string     :description
+      t.string     :measure, :allow_nil => false
+      t.text       :dimensions, :array => true, :allow_nil => false, :default => []
+      t.timestamp  :updated_at
+      t.timestamp  :created_at
+    end
+
+    create_table :showback_events do |t|
+      t.json       :data,       :allow_nil => false, :default => {}
+      t.timestamp  :start_time, :allow_nil => false  # When the event start
+      t.timestamp  :end_time,   :allow_nil => false  # When the event finish
+      t.belongs_to :resource,   :allow_nil => false, :type => :bigint, :polymorphic => true
+      t.json       :context,    :allow_nil => false, :default => {}
```

Comment:
Why not `jsonb`?  From what I've read, there's basically never a reason to use json over jsonb.